package lab6.exceptions;

/**
 * Исключение для случаев, когда скрипт пытается вызвать сам себя
 */
public class SelfCallingScriptException extends RuntimeException {
}
